astropy/io/votable/converters.py

```diff
@@ -609,10 +609,10 @@ def output(self, value, mask):
         value = np.asarray(value)
         mask = np.asarray(mask)
         return ' '.join(base_output(x, m) for x, m in
-                         zip(value.flat, mask.flat))
+                        zip(value.flat, mask.flat))
 
     def binparse(self, read):
-        result = np.fromstring(read(self._memsize),
+        result = np.frombuffer(read(self._memsize),
                                dtype=self._bigendian_format)[0]
```

Comment:
The two uses of np.frombuffer here should be checked to see if it's safe if result a read-only array, (or if something is untested). poke @pllim :wink:

Reply:
Consider me poked. I'll investigate and report back. (Also, whoever wants to be a maintainer for `votable`, feel free to volunteer...) 😉